packages/fxa-auth-server/test/scripts/convert-customers-to-stripe-automatic-tax.ts

```diff
@@ -463,13 +472,45 @@ describe('StripeAutomaticTaxConverter', () => {
         });
       });
 
+      describe("invalid IP address, isn't in same country", () => {
+        beforeEach(async () => {
+          geodbStub.returns({
+            postalCode: 'ABC',
+            countryCode: 'ZZZ',
+          });
+
+          stripeHelperStub.currencyHelper.isCurrencyCompatibleWithCountry.returns(
```

Comment:
`Property 'returns' does not exist on type '(currency:string | null | undefined, country: string | null | undefined) => boolean'. ts(2339) [Ln482, Col 75]`

Fix?:
```
          stripeHelperStub.currencyHelper.isCurrencyCompatibleWithCountry = sinon.stub().returns(
            false
          );
```